src/us/kbase/workspace/database/ByteArrayFileCacheManager.java

```diff
@@ -1,5 +1,7 @@
 package us.kbase.workspace.database;
```

Comment:
This rewrite is so sweeping it might be easier to just look at the new file rather than the diff, or just get a sense of how the old implementation worked from the diff or prior commit.